components/list/list-controls.js

```diff
@@ -37,6 +38,7 @@ export class ListControls extends SelectionControls {
 
 		const parent = findComposedAncestor(this.parentNode, node => node && node.tagName === 'D2L-LIST');
 		if (parent) this._extendSeparator = parent.hasAttribute('extend-separators');
+		if (this._extendSeparator) this.style.setProperty('--d2l-selection-controls-padding', '0px');
```

Comment:
Could this be put in the CSS inside of the `.d2l-list-controls-extend-separator` block that's already defined?

Reply:
That only gets added to the `section`, but I also need it on the `div` that acts as the shadow:
![image](https://github.com/BrightspaceUI/core/assets/13419300/d9e674cf-b006-4030-bf3a-2d852e2503f1)

But if we like that general pattern better, I could create a new class that gets added to both spots?

Reply:
I think that's my preference, to avoid setting CSS variables through JavaScript. But is that gets messy, this is fine.

Reply:
Ok, pushed that change.  It's not quite as minimal as I'd like, I played around with different ways to get the names shorter but it always exposed a bit too much underlying knowledge of the selection controls

Reply:
Chatted with @dbatiste offline about this. We're going to go with the setting the variable in JavaScript for now. The other simple options felt a bit worse (reflecting a state property or passing classes in that exposed too much of how the underlying `selection-controls` worked). We could go with a larger change to refactor the rendering of the controls, something like this:
```
render() {
    return html`
        <div class="extend-separators">this._render()</div>
    `;
}
```
If we decide that's worth doing, that'll be a follow-up PR, keeping this one easier to backport.